<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>528</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>528</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Alphacool LCD test application</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabText">
       <attribute name="title">
        <string>Text</string>
       </attribute>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>401</width>
          <height>201</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxFontHeight">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>100</y>
          <width>71</width>
          <height>29</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>8</number>
        </property>
        <property name="value">
         <number>4</number>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>70</y>
          <width>83</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Font Height:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Time</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLCDNumber" name="lcdNumber">
          <property name="font">
           <font>
            <pointsize>2</pointsize>
           </font>
          </property>
          <property name="digitCount">
           <number>8</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>528</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
